packages/scripts/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## 2.6.0 (Unreleased)
+## 3.0.0 (Unreleased)
+
+### Breaking Changes
+
+- The bundled `eslint` dependency has been updated from requiring `^4.19.1` to requiring `^5.12.1`.
```

Comment:
We should also bump `Babel` at the same time and apply some tweaks to the preset we ship in the same version bump. I will open another PR with changes proposed.